### PR TITLE
CIAC-8501 FortiManager XSIAM Modeling

### PR DESCRIPTION
**Pull request by ellopez777**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-8501](https://jira-dc.paloaltonetworks.com/browse/CIAC-8501)

## Description
The PR contains modeling rules and parsing rules for the FortiManager event log

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FortiManager/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Fortinet FortiManager
+ 
+<~XSIAM>
+ 
+This pack includes Cortex XSIAM content.
+ 
+## Configuration on Server Side
+
+To configure Fortinet FortiManager to forward logs to Cortex XSIAM Broker VM via syslog follow the steps below
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/README.md:9]:
```suggestion
To configure Fortinet FortiManager to forward logs to Cortex XSIAM Broker VM via syslog follow the steps below.
```

File: Packs/FortiManager/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+1. Go to System Settings &rarr Advanced &rarr Syslog Server
+2. On the top pane, select Syslog Server tab 
+3. A new Syslog server window will be open
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/README.md:12]:
```suggestion
1. Go to **System Settings** &rarr **Advanced** &rarr **Syslog Server**.
2. On the top pane, select the **Syslog Server** tab. A new Syslog server window will be open.
```

File: Packs/FortiManager/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+4. Choose a name for the new Syslog server
+5. Insert the IP address (or FQDM - Fully Qualified Domain Names) of the target listener
+6. Insert port number of the target listener (default port is 514)
+for more information [click here - FortiManager documentation](https://docs2.fortinet.com/document/fortimanager/6.0.3/administration-guide/235746/syslog-server)
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/README.md:16]:
```suggestion
3. Choose a name for the new Syslog server.
4. Insert the IP address (or FQDM - Fully Qualified Domain Names) of the target listener.
5. Insert port number of the target listener. (Default port is 514).
For more information see [FortiManager documentation](https://docs2.fortinet.com/document/fortimanager/6.0.3/administration-guide/235746/syslog-server).
```

File: Packs/FortiManager/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    xdm.auth.privilege_level,
+    xdm.target.resource.type,
+    xdm.source.ipv4,
+    xdm.source.ipv6,
+    xdm.event.log_level 
+```
+
+
+
+Severity field according Fortinet FortiManager [Documentation](https://docs.fortinet.com/document/fortimanager/7.4.2/log-message-reference/547625/priority-levels)
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/README.md:62]:
```suggestion
Refer to the severity fields according the Fortinet FortiManager [Documentation](https://docs.fortinet.com/document/fortimanager/7.4.2/log-message-reference/547625/priority-levels).
```

File: Packs/FortiManager/ReleaseNotes/1_0_9.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### New: Fortinet FortiManager Modeling Rule
+
+Added modeling rules for FortiManager event logs via syslog (Available from Cortex XSIAM 2.1).
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/ReleaseNotes/1_0_9.md:6]:
```suggestion
Added modeling rules for FortiManager event logs via syslog. (Available from Cortex XSIAM 2.1).
```

File: Packs/FortiManager/ReleaseNotes/1_0_9.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+#### Parsing Rules
+
+##### New: Fortinet FortiManager Parsing Rule
+
+Added parsing rules for timestamp extraction (Available from Cortex XSIAM 2.1).
```
Comment (ShirleyDenkberg) [inline Packs/FortiManager/ReleaseNotes/1_0_9.md:12]:
```suggestion
Added parsing rules for timestamp extraction. (Available from Cortex XSIAM 2.1).
```

**Comment by ShirleyDenkberg** [comment]
@OBavly @cweltPA Doc review completed.

File: Packs/FortiManager/README.md (hunk 4 above)
Comment (ellopez777) [reply Packs/FortiManager/README.md]: This section was removed

**Review by cweltPA — APPROVED** [review]

**Pull request merged**